pysmt/test/test_euf.py

```diff
@@ -37,6 +37,22 @@ def test_euf(self):
         self.assertSat(check, logic=UFLIRA,
                        msg="Formula was expected to be sat")
 
+    @skipIfNoSolverForLogic(UFLIRA)
+    def test_infix(self):
+        ftype1 = FunctionType(REAL, [REAL])
+        ftype2 = FunctionType(REAL, [REAL, INT])
+        f = Symbol("f", ftype1)
+        g = Symbol("g", ftype2)
+
+        with self.assertRaises(PysmtModeError):
+            f(1.0)
+
+        get_env().enable_infix_notation = True
+        check = Equals(f(1.0), g(2.0, 4))
+
+        self.assertSat(check, logic=UFLIRA,
```

Comment:
We do not need a solver for this. Let's just check that the infix and prefix formula yield the same result.

Reply:
OK